### PR TITLE
skip special punctuation whith italic words

### PR DESCRIPTION
**Pull request by ibrahni**

### CODE REVIEW

File: src/main/java/com/baeldung/common/Utils.java [hunk 1]
```diff
@@ -867,27 +868,20 @@ public  static int getIndexOfFirstTokenStartingWithACharacter(String title) {
         return  Character.isDigit(Character.valueOf(title.charAt(0))) || "Q".equals(String.valueOf(title.charAt(0))) || ">".equals(String.valueOf(title.charAt(0)))? 1 : 0;
      }
 
-    public static boolean isEmpasized(String token, List<String> emphasizedAndItalicTokens) {
+    public static boolean isEmphasized(String token, List<String> emphasizedAndItalicTokens) {
         final String tokenWithoutPossession = token.endsWith(POSSESSION_CHARACTER) ? token.replace(POSSESSION_CHARACTER, EMPTY) : token;
 
-        if (emphasizedAndItalicTokens.contains(removeCommaAtTheEnd(token)) || emphasizedAndItalicTokens.contains(tokenWithoutPossession)) {
+        if (emphasizedAndItalicTokens.contains(removePunctuationAtTheEnd(token, OPTIONAL_PUNCTUATION_IN_WORDS)) || emphasizedAndItalicTokens.contains(tokenWithoutPossession)) {
```
Comment (asif-anwar) [inline src/main/java/com/baeldung/common/Utils.java:874]: Let's pass the _OPTIONAL_PUNCTUATION_IN_WORDS_ as a parameter to this method

File: src/main/java/com/baeldung/common/Utils.java [hunk 2]
```diff
@@ -867,27 +868,20 @@ public  static int getIndexOfFirstTokenStartingWithACharacter(String title) {
         return  Character.isDigit(Character.valueOf(title.charAt(0))) || "Q".equals(String.valueOf(title.charAt(0))) || ">".equals(String.valueOf(title.charAt(0)))? 1 : 0;
      }
 
-    public static boolean isEmpasized(String token, List<String> emphasizedAndItalicTokens) {
+    public static boolean isEmphasized(String token, List<String> emphasizedAndItalicTokens) {
         final String tokenWithoutPossession = token.endsWith(POSSESSION_CHARACTER) ? token.replace(POSSESSION_CHARACTER, EMPTY) : token;
```
Comment (asif-anwar) [inline src/main/java/com/baeldung/common/Utils.java:872]: Let's see if we can handle POSSESSION_CHARACTER also as a special character with our new changes.

File: src/main/java/com/baeldung/common/Utils.java (hunk 1 above)
Comment (ibrahni) [reply src/main/java/com/baeldung/common/Utils.java]: @asif-anwar change done

File: src/main/java/com/baeldung/common/Utils.java (hunk 2 above)
Comment (ibrahni) [reply src/main/java/com/baeldung/common/Utils.java]: @asif-anwar done

**Pull request merged**